.gitlab-ci.yml

```
@@ -321,6 +321,14 @@ cargo-fmt:
   script:
     - cargo +nightly fmt --all -- --check
 
+cargo-clippy:
+  stage:                           test
+  <<:                              *docker-env
+  <<:                              *test-refs
+  script:
+    - SKIP_WASM_BUILD=1 env -u RUSTFLAGS cargo +nightly clippy
```

```suggestion
    - SKIP_WASM_BUILD=1 cargo +nightly clippy
```
no need in removing the variable, it's executed in the reproducible environment.

---

It is getting set earlier but not by docker-env or test-refs from what I can see. For now this is good enough.